content/en/docs/concepts/policy/pid-limiting.md

```diff
@@ -92,8 +92,22 @@ the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
 `SupportPodPidsLimit` to work.
 {{< /note >}}
 
+## PID based eviction
+
+Eviction is the way to configure kubelet to start terminating a pod if it misbehaving.
```

Comment:
"Eviction is the way to configure kubelet ..." this statement may confuse readers ... please consider a revision.